src/puppetlabs/trapperkeeper/authorization/acl.clj

```diff
@@ -125,8 +129,13 @@
 (schema/defn ^:always-validate new-domain :- ACE
   "Creates a new ACE for a domain"
   [auth-type :- AuthType
-   {:keys [certname extensions]} :- ACEConfig]
+   {:keys [certname extensions rbac]} :- ACEConfig]
   (cond
+    (map? rbac)
```

Comment:
every other stanza here has a comment, this one seems self explanatory but might be best for consistency

Reply:
I think the comment should go one line up.